File: Sources/SPFNCore/SPFNVersion.swift
// SPFN Mobile — Step 1 scaffold. No behaviour, no release promise.

/// The single mobile release-train version.
///
/// `SPFNVersion.current` must equal the repository `VERSION` file byte-for-byte
/// (trimmed). `tools/validate/validate.sh` and `SPFNCoreTests` both enforce this.
public enum SPFNVersion
{
    /// Release-train start version per decision D9 (2026-08-01). Pre-release:
    /// nothing has been tagged or published, and 1.0.0 waits on Step 5 evidence.
    public static let current: String = "0.1.0-alpha.3"
}

File: android/spfn-core/src/main/kotlin/xyz/superfunction/spfn/core/SpfnVersion.kt
// SPFN Mobile — the single mobile release-train version.
//
// Counterpart of Sources/SPFNCore/SPFNVersion.swift. One constant in one file because
// three other things are held to it: `tools/validate/validate.sh` greps this path for the
// contents of VERSION, SpfnCoreTest reads VERSION off disk and compares, and
// gradle.properties carries the same string for publication.

package xyz.superfunction.spfn.core

/** Mirror of the Swift `SPFNVersion`. Must equal the repository VERSION file. */
object SpfnVersion
{
    const val CURRENT: String = "0.1.0-alpha.3"
}
